src/conn/conn_log.c

```diff
@@ -104,19 +104,20 @@ __logmgr_version(WT_SESSION_IMPL *session, bool reconfig)
      * does not mean the direction of change from the release we may be running currently.
      */
     if (conn->compat_major < WT_LOG_V2_MAJOR) {
-        new_version = 1;
         first_record = WT_LOG_END_HEADER;
+        new_version = 1;
```

Comment:
Is there a reason for this line movement?

Reply:
My reason for the line movement was to keep the ordering of variable assignment consistent. Let me know if that's not allowed and I'll change it back.